.github/workflows/release.yaml

```
@@ -14,43 +14,19 @@ on:
           - major
 
 jobs:
-  release-to-pypi:
-    uses: bakdata/ci-templates/.github/workflows/python-poetry-release.yaml@v1.18.2
+  create-github-release-push-tag:
+    runs-on: ubuntu-22.04
+    uses: bakdata/ci-templates/.github/workflows/python-poetry-release.yaml@v1.23.0
+    name: build project and create GitHub release
```

this doesn't "build the project"
```suggestion
    name: Release
```